src/libsync/abstractnetworkjob.cpp

```diff
@@ -374,8 +374,8 @@ void AbstractNetworkJob::abort()
         // calling abort will trigger the execution of finished()
         // with _reply->error() == QNetworkReply::OperationCanceledError
         // the api user can then decide whether to discard this job or retry it.
-        _reply->abort();
         _aborted = true;
+        _reply->abort();
```

Comment:
Since the order seems to have been the problem, I'd like to have a comment explaining this to future readers.